NumberBaseball/NumberBaseball/main.swift

```diff
@@ -76,13 +142,27 @@ func checkStrike(_ strike: Int, _ strikeCount: Int = 3) -> Bool {
     }
 }
 
-func addScore(to resultStrikeAndBall: inout [Int], computer: Array<Int>.Index, user: Array<Int>.Index) {
+func calculateResult(computerNumbers: [Int], userNumbers: [Int]) -> [Int] {
+    
+    var resultStrikeAndBall: [Int] = [0, 0]
+    
+    for userIndex in 0...(userNumbers.count - 1) {
+        
+        guard let computerIndex = computerNumbers.firstIndex(of: userNumbers[userIndex]) else { continue }
+        
+        addScore(to: &resultStrikeAndBall, computerIndex: computerIndex, userIndex: userIndex)
+    }
+    
+    return resultStrikeAndBall
+}
+
+func addScore(to resultStrikeAndBall: inout [Int], computerIndex: Int, userIndex: Int) {
```

Comment:
```suggestion
func addScore(to resultStrikeAndBall: inout [Int],
              computerIndex: Int,
              userIndex: Int) {
```
파라미터가 많은 경우엔 이런식으로 줄바꿈을 해주면 가독성이 조금 올라갑니다!
저는 개인적으로 3개부터는 줄바꿈을 해주는 편입니다😊

Reply:
앗... 아직 파라미터 줄 바꿈이 익숙치 않아서 적용을 못했던 것 같습니다.
마이노의 조언을 듣고 줄 바꿈을 해보니까 가독성이 훨씬 좋아졌어요!! ☺️

저희가 캐치하지 못한 부분을 하나하나 링크까지 상세하게 설명해주셔서 정말 감동이었습니다...! 
앞으로도 마이노에게 많이 배우고싶어요!! 🥹
감사합니다 🙏